package.json

```diff
@@ -34,6 +33,7 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.22.5",
     "@nrwl/nx-cloud": "^15.3.5",
+    "@types/estree": "^1.0.1",
```

Comment:
this is to avoid typescript errors from `@types/eslint` package for which outdated `@types/estree` peer dependency will be installed otherwise

better fix, maybe as a next step would be to update `dts-cli`